--- gcc-5.2.0/gcc/cp/except.c	2015-02-12 15:38:33.000000000 +0000
+++ patches/gcc-5.2.0/gcc/cp/except.c.fix	2023-12-09 23:23:36.040350952 +0000
@@ -1040,7 +1040,8 @@
      unless the system headers are playing rename tricks, and if
      they are, we don't want to be confused by them.  */
   id = DECL_NAME (fn);
-  return !!libc_name_p (IDENTIFIER_POINTER (id), IDENTIFIER_LENGTH (id));
+  return !!libc_name::libc_name_p (IDENTIFIER_POINTER (id),
+				   IDENTIFIER_LENGTH (id));
 }
 
 /* Returns nonzero if an exception of type FROM will be caught by a
