--- gcc-5.4.0/libgcc/config/i386/linux-unwind.h	2015-01-05 12:33:28.000000000 +0000
+++ patches/gcc-5.4.0/libgcc/config/i386/linux-unwind.h.fix	2023-12-09 18:55:54.980163780 +0000
@@ -58,7 +58,7 @@
   if (*(unsigned char *)(pc+0) == 0x48
       && *(unsigned long long *)(pc+1) == RT_SIGRETURN_SYSCALL)
     {
-      struct ucontext *uc_ = context->cfa;
+      struct ucontext_t *uc_ = context->cfa;
       /* The void * cast is necessary to avoid an aliasing warning.
          The aliasing warning is correct, but should not be a problem
          because it does not alias anything.  */
