--- gcc-4.7.4/gcc/doc/invoke.texi	2024-01-06 12:15:50.809414170 +0000
+++ patches/gcc-4.7.4/gcc/doc/invoke.texi.fix	2024-01-06 12:14:36.727491924 +0000
@@ -5352,7 +5352,7 @@
 Dump after delayed branch scheduling.
 
 @item -fdump-rtl-dce1
-@itemx -fdump-rtl-dce2
+@item -fdump-rtl-dce2
 @opindex fdump-rtl-dce1
 @opindex fdump-rtl-dce2
 @option{-fdump-rtl-dce1} and @option{-fdump-rtl-dce2} enable dumping after
