--- gcc-4.7.0/gcc/cp/cfns.h	2009-04-21 20:03:23.000000000 +0100
+++ patches/gcc-4.7.0/gcc/cp/cfns.h.fix	2024-01-06 17:37:10.964790685 +0000
@@ -53,6 +53,9 @@
 static unsigned int hash (const char *, unsigned int);
 #ifdef __GNUC__
 __inline
+#ifdef __GNUC_STDC_INLINE__
+__attribute__ ((__gnu_inline__))
+#endif
 #endif
 const char * libc_name_p (const char *, unsigned int);
 /* maximum key range = 391, duplicates = 0 */
