--- gcc-4.8.3/gcc/cp/except.c	2023-12-13 22:11:13.149566144 +0000
+++ patches/gcc-4.8.3/gcc/cp/except.c.fix	2023-12-13 22:13:21.166397149 +0000
@@ -1025,7 +1025,8 @@
      unless the system headers are playing rename tricks, and if
      they are, we don't want to be confused by them.  */
   id = DECL_NAME (fn);
-  return !!libc_name_p (IDENTIFIER_POINTER (id), IDENTIFIER_LENGTH (id));
+  return !!libc_name::libc_name_p (IDENTIFIER_POINTER (id),
+				   IDENTIFIER_LENGTH (id));
 }
 
 /* Returns nonzero if an exception of type FROM will be caught by a
