--- gcc-4.9.2/gcc/cp/except.c	2014-01-02 22:23:26.000000000 +0000
+++ patches/gcc-4.9.2/gcc/cp/except.c.fix	2023-12-10 15:37:17.449661220 +0000
@@ -1030,7 +1030,8 @@
      unless the system headers are playing rename tricks, and if
      they are, we don't want to be confused by them.  */
   id = DECL_NAME (fn);
-  return !!libc_name_p (IDENTIFIER_POINTER (id), IDENTIFIER_LENGTH (id));
+  return !!libc_name::libc_name_p (IDENTIFIER_POINTER (id),
+				   IDENTIFIER_LENGTH (id));
 }
 
 /* Returns nonzero if an exception of type FROM will be caught by a
